### DISCUSSION TITLE
Wazuh full featured?

### DISCUSSION THREAD

**Discussion by atndon** [Security-Onion-Solutions/securityonion#7744] [General] [answered]
Hi,

Thank you for the amazing security onion platform.

I know the platform is shipped with Wazuh server included. When I read the documentation and the ossec.conf file I can see many of Wazuh features is not included or disabled. My question here is, can I enable these features? If yes, is the next step to create dashboards to visualize the data?

Thank you for the answer in advance.

**Comment by dougburks** [comment, marked as answer]
In general, yes, you can enable additional features. If there are not already Kibana dashboards or Hunt queries for the relevant data, then you can create your own.
